tests/cli-misc.rs

```diff
@@ -568,6 +569,33 @@ fn rename_rls_remove() {
     });
 }
 
+#[test]
+fn install_profile() {
+    let temp_dir = tempfile::Builder::new()
+        .prefix("fakebin")
+        .tempdir()
+        .unwrap();
+    let temp_dir_path = temp_dir.path().to_str().unwrap();
+
+    setup(&|config| {
+        let args: Vec<&str> = vec!["-y", "--profile", "minimal"];
+        run(
+            config,
+            "rustup-init",
+            &args,
+            &[
+                ("RUSTUP_INIT_SKIP_PATH_CHECK", "no"),
+                ("PATH", &temp_dir_path),
+            ],
+        );
+
+        expect_component_executable(config, "rustup");
+        expect_component_executable(config, "rustc");
+        // TODO failing
+        expect_component_not_executable(config, "cargo");
```

Comment:
Minimal surely includes cargo?  Perhaps expect `rustfmt` to not be there?

Reply:
We generally don't make the tests match the downloads exactly, and it was hard to get this to work with rustfmt, etc., because they are not present in the mock manifest

Reply:
Aaah okay, I misunderstood, that sounds fine then.